alfresco-dynamic-extensions-repo/build.gradle

```diff
@@ -1,6 +1,13 @@
 subprojects {
     group "eu.xenit"
 
+    sonarqube {
+        properties {
+            property "sonar.projectName", project.name
+            property "sonar.projectKey", "${project.parent.name}:${project.name}"
```

Comment:
`projectKey` identifies the project on sonarcloud. It should be the same for all projects, or it will generate a lot of different projects

Reply:
@vierbergenlars Ready for review